pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "podaac-concise"
-version = "0.3.0-alpha.5"
+version = "0.3.0"
```

Comment:
Leave the -alpha.x part in the version. The github workflows take care of the versioning for us.

Reply:
`git rebase develop` is stuck in an endless loop for some reason. If I can't get it straightened out I'll create a new branch off of 'develop' and open a new PR.